### DISCUSSION TITLE
Module not found: Error: Can't resolve '@theme/Seo'

### DISCUSSION THREAD

**Discussion by rafaelrdealmeida** [facebook/docusaurus#7351] [Q&A] [answered]
Hi,

We are testing some customizations on the blog and for that we created a specific branch called `test_blog`. The customizations in this branch work fine, but when we rebase `teste_blog` and merge into main we get the following error in both `main` and `teste_blog`:

```
Module not found: Error: Can't resolve '@theme/Seo' in '/workspace/support/website/src/theme/BlogPostItem'
client (webpack 5.72.0) compiled with 1 error
```

- we are on docusaurus beta.20
- repo: https://gitlab.com/unesp-labri/sites/apoio

Any idea what's going on?

best

**Comment by slorber** [comment, marked as answer]
You have swizzled components there. As always, take a look at those and make sure to update their code according to latest Docusaurus version you are using.

In this specific case, the `@theme/Seo` component does not exist anymore
It has been replaced by `import { PageMetadata } from '@docusaurus/theme-common';`

But you can't replace it like that and expect your site to work: you really have to check and eventually re-swizzle the components on upgrades to make sure your swizzled components are fine. In this case you'll notice that our official `BlogPostItem` does not even use `PageMetadata` or `Seo` anymore (as it was moved to another component). 

Remember that if you have customization use-cases, you can report them here so that we make them less fragile over time: https://github.com/facebook/docusaurus/discussions/5468


---

I'd recommend that you re-read the swizzle doc to understand better all this. Also when doing customizations on 2 branches, You'd rather use the same Docusaurus version across the branches

**Reply by rafaelrdealmeida** [reply]
thanks for the reply and instructions!

I make the adjustments and the blog page now works in both branches (`main` and `teste_blog`), but in `main` the containers are unconfigured. Already in `teste_blog` are ok.

I'm not sure if the problem is in my css and some configuration of docusuaurus that I let go. Could you tell me?

see: https://apoio.labriunesp.org/blog

best

**Reply by slorber** [reply]
I can't dedicate much time to help debug site specific problems: we have a discord server and many active members could help you there

> but in main the containers are configured

I don't even know what that means


---

When in trouble, the solution is always to re-swizzle everything and start from 0 customization. 

If you were to customize the first time, you should be able to re-customize the 2nd time

I know it's annoying but I can't spend my time doing this for you. 

The best is that you help us make this process less painful over time
